utils/testWrapper/index.js

```diff
@@ -54,11 +54,25 @@ const getLongKey = () => {
   return key;
 };
 
+const gracefulShutDown = async () => {
+
+  const environmentManagerFactory = require('../../environmentManager');
+  if (environmentManagerFactory && environmentManagerFactory.hasInstance()) {
+    await environmentManagerFactory.destroy();
+  }
+
+  const impressionManagerFactory = require('../../listener/manager');
+  if (impressionManagerFactory.hasInstance && impressionManagerFactory.hasInstance()) {
```

Comment:
Idem. `if (impressionManagerFactory.hasInstance()) {` should be enough since `impressionManagerFactory` is an object with a hasInstance method.